### DISCUSSION TITLE
HDMI to component converter

### DISCUSSION THREAD

**Discussion by regmar505** [GerryDazoo/Slinger#492] [Q&A]
At my remote location, I'd like to bring one of my two Slingbox M2 but the Set Top Box there provides only HDMI output. I wonder if someone tried a HDMI to component converter, if so please post a link for your model and let me know whether you can get 1080i.

**Comment by Dhextreme** [comment]
Same for me, only HDMI so I use this model and it work fine. https://www.amazon.ca/-/fr/dp/B07VGWL5P1?psc=1&ref=ppx_yo2ov_dt_b_product_details . It have a resolution button on it. I can't find better. I use the component output  to set the res because it's not very easy doing that on the sling player.

**Reply by regmar505** [reply]
Thank you for the link. May I ask you whether you can get a resolution of 1080i? What do you mean by "I can't find better"?

**Reply by Dhextreme** [reply]
You're welcome! Yes it as 1080i, that's the one I use. I like some quality on what I buy but I can't find a better one, I mean made in USA , not China stuff but it do the job as expected.
